notebooks/features/cognitive_services/CognitiveServices - OpenAI Embedding.ipynb

```diff
@@ -74,6 +74,10 @@
     "deployment_name_embeddings = \"text-search-ada-doc-001\"\n",
     "deployment_name_embeddings_query = \"text-search-ada-query-001\"\n",
     "\n",
+    "# If available, try using the latest embedding model: https://openai.com/blog/new-and-improved-embedding-model\n",
```

Comment:
Can we just switch this over to this directly and keep the old one in comments?

Reply:
done.